client/my-sites/domains/domain-management/settings/cards/name-servers-card.tsx

```diff
@@ -239,7 +239,9 @@ const NameServersCard = ( {
 				{ nameservers.map( ( nameserver ) => (
 					<p key={ nameserver }>{ nameserver }</p>
 				) ) }
-				<Button onClick={ editCustomNameServers }>Edit custom name servers</Button>
+				<Button onClick={ editCustomNameServers }>
+					{ translate( 'Edit custom name servers' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Use custom name servers:' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/i18n-custom-nameserver-missing-translation)